ama-sample.conf

```diff
@@ -1,17 +1,24 @@
 {
+  #required
   pg => 'postgresql://mojo:mojo@/test1',
+  #recommended
```

Comment:
Make a note to say that the secret should be changed from mojo to something unique.
You might even reference the [Mojo doc](http://mojolicious.org/perldoc/Mojolicious#secrets).